### PR TITLE
[HPRO-692] Fix patient status bulk import issue

### PR DESCRIPTION
**Pull request by shyamnaru**
This PR fixes combination of duplicate participant_id and organization insertions in the patient_status table

### CODE REVIEW

**Review by stephenyeargin — APPROVED** [review]

**Review by jt2k — COMMENTED** [review]
Should we also add a check to the import upload? We could make it so that uploads with multiple rows with the same participant ID are treated as invalid and not processed.

**Comment by shyamnaru** [comment]
Yes, we could throw a form error when the import file contains duplicate participant ids.

File: src/Pmi/Controller/DefaultController.php [hunk 1]
```diff
@@ -447,15 +447,15 @@ public function participantAction($id, Application $app, Request $request)
...
                     'participant_id' => $id,
                     'organization' => $app->getSiteOrganizationId()
-                ]);
+                ], ['history_id' => 'desc'], 1);
                 if (!empty($patientStatusData) && empty($patientStatusForm['comments']->getData())) {
                     $patientStatusForm['comments']->addError(new FormError('Please enter comment'));
                 }
                 if ($patientStatusForm->isValid()) {
-                    $patientStatusId = !empty($patientStatusData) ? $patientStatusData['id'] : null;
+                    $patientStatusId = !empty($patientStatusData) ? $patientStatusData[0]['id'] : null;
```
Comment (jt2k) [inline src/Pmi/Controller/DefaultController.php:458]: Now that we're adding the unique constraint and cleaning up the data, do we still need these changes?

Comment (shyamnaru) [reply src/Pmi/Controller/DefaultController.php:458]: Yes, we don't need them, I'll update. Thanks!

**Review by stephenyeargin — APPROVED** [review]

**Review by jt2k — APPROVED** [review]

**Pull request merged**